playgrounds/react/resources/js/Components/Layout.tsx

```diff
@@ -5,7 +5,7 @@ export default function Layout({ children }) {
 
   return (
     <>
-      <nav className="flex items-center space-x-6 bg-slate-800 px-10 py-6 text-white">
+      <nav className="sticky top-0 flex items-center space-x-6 bg-slate-800 px-10 py-6 text-white">
```

Comment:
allows for reproduction to be tested -- you need to be able to click a link while in a scrolled-down state